code-orange/DrawerView.swift

```diff
@@ -44,20 +41,20 @@ class DrawerView: UIView {
   }
   
   private func commonInit() {
-    layer.masksToBounds = true
+    layer.masksToBounds = false
+    layer.shadowColor = UIColor.black.cgColor
     layer.cornerRadius = 24
-    backgroundColor = .white
-    alpha = 0 // TODO: use hiding constraint
+    layer.shadowOpacity = 0.5
+    layer.shadowOffset = .zero
+    layer.shadowRadius = 12
```

Comment:
I added a shadow infra (copied from nexar) in the file "Buttons" you are welcome to use it here

Reply:
I'll add usage for it

Reply:
done 🚀